CP05MOAS-A6263/CP05MOAS-A6263_R00005_ingest.csv

```diff
@@ -1,8 +1,8 @@
-#mi.dataset.driver.auv_eng.auv.auv_eng_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/*.csv,CP05MOAS-A6263-00-ENG000000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
-#mi.dataset.driver.flort_kn.auv.flort_kn_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/*.csv,CP05MOAS-A6263-01-FLORTN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
-#mi.dataset.driver.dosta_ln.auv.dosta_ln_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/*.csv,CP05MOAS-A6263-02-DOSTAN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
-#mi.dataset.driver.ctdav_n.auv.ctdav_n_auv_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/*.csv,CP05MOAS-A6263-03-CTDAVN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
-#mi.dataset.driver.nutnr_n.nutnr_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/*.SUN,CP05MOAS-A6263-04-NUTNRN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
-#mi.dataset.driver.adcpa_n.adcpa_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/*.ADC,CP05MOAS-A6263-05-ADCPAN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.auv_eng.auv.auv_eng_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/*.txt,CP05MOAS-A6263-00-ENG000000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
+#mi.dataset.driver.flort_kn.auv.flort_kn_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/*.txt,CP05MOAS-A6263-01-FLORTN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
+#mi.dataset.driver.dosta_ln.auv.dosta_ln_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/*.txt,CP05MOAS-A6263-02-DOSTAN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
+#mi.dataset.driver.ctdav_n.auv.ctdav_n_auv_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/*.txt,CP05MOAS-A6263-03-CTDAVN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
+#mi.dataset.driver.nutnr_n.nutnr_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/DATA/*.SUN,CP05MOAS-A6263-04-NUTNRN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
+#mi.dataset.driver.adcpa_n.adcpa_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/ADCP/*.ADC,CP05MOAS-A6263-05-ADCPAN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
 #mi.dataset.driver.parad_n.auv.parad_n_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/*.csv,CP05MOAS-A6263-06-PARADN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
```

Comment:
All of these are commented out, so it doesn't really matter... but why does line 8 still have *.csv?

Reply:
Agreed, it should be *.txt. Fixed that

Reply:
Looks good.